src/integration/__tests__/integration.test.ts

```diff
@@ -11,15 +11,21 @@ if (!isOnCI) {
 } else {
   // These tests ensure that *any* query works against our express app.
   const request = require("supertest")
-  const app = require("../../index")
-  const gql = require("lib/gql")
+  const app = require("../../index").default
+  const gql = require("lib/gql").default
```

Comment:
This wasn’t working, so I’m unsure how this was passing CI.

Reply:
I can only imagine this env var isn’t set? 🤷‍♂️  https://github.com/artsy/metaphysics/pull/1660/files#diff-2dbea2b3cd7165fd7cc283ff39969a84R5